cobbler/actions/buildiso/netboot.py

```diff
@@ -377,7 +377,7 @@ def generate_profile(self, distro_breed: str) -> str:
         :param distro_breed: The name of the distribution breed.
         :return: The generated append line.
         """
-        self.append_line = " append initrd=%s.img" % self.distro_name
+        self.append_line = "  APPEND initrd=%s.img" % self.distro_name
```

Comment:
are 2 leading spaces intended?

Reply:
Yes. The reason for this is that we want that to appear correctly in the corresponding configuration file. This is also the reason why this was uppercased.